api/preprints/serializers.py

```diff
@@ -205,70 +200,15 @@ class PreprintSerializer(TaxonomizableSerializerMixin, MetricsSerializerMixin, J
         },
     )
 
-    has_coi = DisableIfSwitch(
```

Comment:
`DisableIfSwitch` reffers to the waffle switches that turn the field on. The study structure made it so we had to flip features on individually.